src/utils/shallowEqual.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-self-compare */
+
+function is(x: any, y: any) {
+  if (x === y) {
+    return x !== 0 || y !== 0 || 1 / x === 1 / y;
+  } else {
+    return x !== x && y !== y;
+  }
+}
+
+export function shallowEqual(objA: any, objB: any) {
```

Comment:
This is the implementation from react-redux. Probably overkill though.
Maybe we should use an external lib for that so there's a *chance* of not duplicating code when another lib in the bundle uses the same helper?

Reply:
@markerikson do you have an opinion there?

Reply:
Kind of the age-old question: rely on a dependency and add to the potential deps size, or copy it into your codebase and maybe end up with duplicate code?

I don't have strong feelings on the "overkill" part.

I'd be inclined to have a copy-pasted version because we need it here, Redux doesn't have one, we can't import it from React-Redux even if R-R did export it, etc.